Cargo.toml

```
@@ -19,6 +19,7 @@ default = ["bevy", "sled"]
 bevy = ["dep:bevy_ecs"]
 rocksdb = ["dep:rocksdb"]
 sled = ["dep:sled"]
+pickledb=["dep:pickledb"]
```

```suggestion
pickledb = ["dep:pickledb"]
```
 Please follow surrounding whitespace style, repeats.